README.md

```diff
@@ -75,12 +75,16 @@ a suite file defines a series of tests to run.
 #### suite file format
 
 ```ruby
+# format description:
 # each test can be executed by either target duration using :time => N secs
 # or by number of events with :events => N
 #
+# you can specify the number of filter worker threads for each test with :workers => N
+# if you don't specify workers, it defaults to 1
```

Comment:
I didn't realize 2.0 changed that behavior. Honestly, I would expect not specifying workers to do whatever logstash does by default when using it directly. So for 1.x that was just defaulting to 1 thread, but if 2.0 defaults to half the amount of cpu cores, I would expect that to happen when running lsperfm. 

But that's just me, if you prefer having it always default to one, then I'm cool with that. If so, then yeah, we should note the behavior in a comment.

If we do decide to just have it default to whatever logstash does, then I guess the way to go about it is just construct the command send to popen to not have the -w if workers is never specified in the test.